pkg/kube/inject/testdata/inject/user-volume.yaml.injected

```diff
@@ -86,7 +86,7 @@ spec:
               fieldPath: metadata.labels['service.istio.io/canonical-revision']
         - name: PROXY_CONFIG
           value: |
-            {}
+            {"proxyMetadata":{"DNS_AGENT":""}}
```

Comment:
This change is because the old test was using DefaultMeshConfig whereas the new test actually gets the mesh config a user will get with the default profile